lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -68,21 +77,30 @@ export class PublicClientApplication implements IPublicClientApplication {
      * @param IController Optional parameter to explictly set the controller. (Will be removed when we remove public constructor)
      */
     public constructor(configuration: Configuration, controller?: IController) {
+        this.configuration = configuration;
         if (controller) {
             this.controller = controller;
         } else {
-            const standardOperatingContext = new StandardOperatingContext(
-                configuration
+            const operatingContext = new UnknownOperatingContext(configuration);
```

Comment:
Would this not be a breaking change? How do existing consumers of the library calling `new PublicClientApplication(configuration)` get the standard controller?

Reply:
They don't ... why would someone want a specific implementation?  That's the point of the controller interface.